src/librustdoc/html/static/main.js

```diff
@@ -450,7 +450,7 @@ function defocusSearchBar() {
                 set_fragment(cur_line_id);
             }
         }
-    })();
+    }());
```

Comment:
While this isn't bad, it seems spurious in a "move code" commit rather than a cleanup one.  You can just resolve this if you don't care, otherwise move it to a different commit when you next rebase.